core/templates/dev/head/forms/formBuilder.js

```diff
@@ -559,6 +561,9 @@ oppia.factory('rteHelperService', [
           RTE_COMPONENT_SPECS[componentId].customization_arg_specs),
         name: RTE_COMPONENT_SPECS[componentId].frontend_name,
         iconDataUrl: RTE_COMPONENT_SPECS[componentId].icon_data_url,
```

Comment:
Can we get rid of this? Then every component just has a single preview url, which seems cleaner.

(Having said that, how do you handle block vs inline display?)